website_sale_product_brand/views/product_brand.xml

```diff
@@ -6,10 +6,10 @@
     <template id="product_brands" name="Brands">
         <t t-call="website.layout">
             <t t-set="additional_title">Product Brands</t>
-            <div id="wrap" t-attf-class="wrapper"> 
+            <div id="wrap" t-attf-class="wrapper">
```

Comment:
this is not a [format string](https://www.odoo.com/documentation/8.0/glossary.html#term-format-string) you may use `t-att-class="wrapper"`.